jujucrashdump/crashdump.py

```diff
@@ -52,8 +52,10 @@
 
 TAR_CMD = """sudo find {dirs} -mount -type f -size -{max_size}c -o \
 -size {max_size}c 2>/dev/null | sudo tar -pcf /tmp/juju-dump-{uniq}.tar \
---files-from - 2>/dev/null; sudo tar --append -f /tmp/juju-dump-{uniq}.tar \
--C /tmp/{uniq}/addon_output . || true"""
+--files-from - 2>/dev/null; sudo netstat -taupn | grep LISTEN 2>/dev/null > \
```

Comment:
I don't think a collect operation should be injected into the tar command.  Also, if the `grep LISTEN` fails to match, it will exit 1.

Reply:
the grep LISTEN failing and exiting means we wouldn't append the `listening.txt` to the tar, but the whole TAR_CMD is a separated list of commands.

Regarding putting that command in there, currently, juju-crashdump will only run commands on the unit in two places, via addons and via that command. The only thing that comes back form the node is the tarfile created in this command.

An alternative approach to this would be to add a helper function to run additional commands on the machines and put their files in a known location, similar to the `/tmp/{uniq}/addon_output` location.

Reply:
I'm slightly incorrect, `grep` (or `netstat`) returning non-zero just ends with an empty `/tmp/{uniq}/listening.txt`

I'm working on a change that will allow a more clear path to managing commands to run, as juju-crashdump does not currently have a way to collect output of commands on units.